README.md

```diff
@@ -52,4 +58,43 @@ Prerequisites:
     * Enter password
 * `python manage.py runserver`
 * Go to http://localhost:8000/staff
-* Sign in as the superuser you created above
\ No newline at end of file
+* Sign in as the superuser you created above
+
+### Note for PyCharm Users (All Platforms)
+If you want to use `badgr-server` in PyCharm, you have to add the `./apps` directory to the local Python path. You can do this by going to Settings->Project->Project Interpreter and clicking on the gear icon next to the selected interpreter. Then go to Show All. With the project interpreter selected, the bottom icon in the right will open the Python path. Click the "+" icon and select the `./apps` folder to add to the path.
```

Comment:
Please name the "Bottom icon in the right" or the window that it should open. Otherwise we'll have a bad time if IntelliJ decides to change the order.